app/src/sharedTest/java/org/oppia/android/app/databinding/TextViewBindingAdaptersTest.kt

```diff
@@ -189,7 +189,7 @@ class TextViewBindingAdaptersTest {
   }
 
   @Test
-  fun testTextViewBindingAdapters_ltrIsEnabled_port_profileLastVisitedTextIsCorrectCase5() {
+  fun testSetProfileLastVisitedText_forTimeTwoHourAgo_setsLastUsedTwoHoursAgoText() {
```

Comment:
Nit: ``TwoHoursAgo``. Ditto elsewhere (I think there are some pluralization issues in other tests).

Reply:
Done.